dotcom-rendering/cypress/e2e/parallel-7/snapshot-standard-layout.e2e.cy.js

```diff
@@ -0,0 +1,23 @@
+import { disableCMP } from '../../lib/disableCMP.js';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+import { breakpoints } from '@guardian/source-foundations';
+import '@percy/cypress';
+
+describe('Snapshot standard layout', function () {
+	beforeEach(function () {
+		disableCMP();
+		setLocalBaseUrl();
+	});
+
+	it('Snapshot all breakpoints', function () {
+		cy.visit(
+			'/Article?url=https://www.theguardian.com/uk-news/2020/dec/04/edinburgh-hit-by-thundersnow-as-sonic-boom-wakes-residents',
+		);
+
+		cy.hydrate();
+
+		cy.percySnapshot('Standard Layout', {
+			widths: Object.values(breakpoints),
```

Comment:
Assuming we're going to be mapping all variations, would a subset of breakpoints make more sense here? It's very much a trade off but this could be a good place to reduce coverage in favour of speed, cost, etc.

Reply:
👍